models/dataset_test.go

```diff
@@ -1103,13 +1103,14 @@ func TestVersionLinksDeepCopy(t *testing.T) {
 					HRef: "versionHRef",
 				},
 			})
-			So(vl2, ShouldNotEqual, vl)
-			So(vl2.Dataset, ShouldNotEqual, vl.Dataset)
-			So(vl2.Dimensions, ShouldNotEqual, vl.Dimensions)
-			So(vl2.Edition, ShouldNotEqual, vl.Edition)
-			So(vl2.Self, ShouldNotEqual, vl.Self)
-			So(vl2.Spatial, ShouldNotEqual, vl.Spatial)
-			So(vl2.Version, ShouldNotEqual, vl.Version)
+
+			So(vl2, ShouldNotBeNil)
+			So(vl2.Dataset, ShouldNotBeNil)
+			So(vl2.Dimensions, ShouldNotBeNil)
+			So(vl2.Edition, ShouldNotBeNil)
+			So(vl2.Self, ShouldNotBeNil)
+			So(vl2.Spatial, ShouldNotBeNil)
+			So(vl2.Version, ShouldNotBeNil)
```

Comment:
Are these equivalent tests? Should these not be fixed some other way so that the specific values are still tested rather than just accepting any value as long as it isn't nil?

Reply:
These tests have now been updated.